protocol/chat.go

```diff
@@ -207,18 +204,23 @@ func CreateOneToOneChat(name string, publicKey *ecdsa.PublicKey) Chat {
 }
 
 func CreatePublicChat(name string) Chat {
+	rand.Seed(time.Now().Unix())
```

Comment:
Isn't it enough to call it once or don't call it at all? Not sure what's the penalty hit for doing it over and over in `CreatePublicChat`.

Reply:
I think call it once at program start is enough, you are right.

I believe if you don't call it, it will always have the same seed, so the chats will always be created with the same color order, and you'd be hitting the first colors much more.